tests/test_models_interfaces.py

```diff
@@ -220,14 +221,17 @@ def test_encoding(backend, frontend, layer, seed):
 
 
 @pytest.mark.parametrize("layer,seed", zip(DECODING_LAYERS, [1, 2, 1, 1]))
-@pytest.mark.parametrize("analytic", [True, False])
-def test_decoding(backend, frontend, layer, seed, analytic):
-    if frontend.__name__ == "qiboml.models.keras":
+def test_decoding(backend, frontend, layer, seed):
+    if frontend.__name__ == "qiboml.interfaces.keras":
         pytest.skip("keras interface not ready.")
     if backend.name not in ("pytorch", "jax"):
         pytest.skip("Non pytorch/jax differentiation is not working yet.")
-    if analytic and not layer is dec.Expectation:
+    if layer.analytic and not layer is dec.Expectation:
         pytest.skip("Unused analytic argument.")
```

Comment:
```suggestion
```

this is not needed anymore I believe, as `analytic` is not a pytest parameter anymore